bibtasklets/bst_orcidsync.py

```diff
@@ -49,6 +64,25 @@ def orcid_data_to_sync():
             yield orcid.decode('utf8'), token.decode('utf8'), user_info['email'].decode('utf8'), name
 
 
+def count_orcid_data_to_sync():
+    """
+    Count the number of ORCID tokens (and relative data) to be synch'ed (and
+    thus sent to Redis). This is used at Lab's side to check weather the right
+    number of token has been processed.
+
+    Returns (int): a number.
+    """
+    query = """
+        SELECT count(*) FROM (
+            SELECT personid, count(*) AS cnt FROM inspirehep.aidPERSONIDDATA WHERE tag='uid' AND personid IN (
```

Comment:
using a hardcoded `databasename` is not advised. while I understand the motivation to fully specify db+table the db name is configurable in `invenio.conf`. On my test install it's `inspiredemo` for example, or `inspiredev`. you get the idea. `run_sql` reads the proper db name from config, so you shouldn't specify it at all for portability.

Reply:
Fixed (I forgot to remove the db name when I copy/pasted from my client)